pkg/certs/certs.go

```diff
@@ -74,12 +76,22 @@ func (c *Certs) watch() (err error) {
 		}
 	}()
 
-	if err = notify.Watch(c.certFile, c.e, eventWrite...); err != nil {
-		return err
-	}
-
-	if err = notify.Watch(c.keyFile, c.e, eventWrite...); err != nil {
-		return err
+	// Windows doesn't allow for watching file changes but instead allows
+	// for directory changes only.
+	if runtime.GOOS == "windows" {
+		if err = notify.Watch(filepath.Dir(c.certFile), c.e, eventWrite...); err != nil {
+			return err
+		}
+		if err = notify.Watch(filepath.Dir(c.keyFile), c.e, eventWrite...); err != nil {
+			return err
+		}
+	} else {
+		if err = notify.Watch(c.certFile, c.e, eventWrite...); err != nil {
```

Comment:
I tested in Linux, it will still work if you put filepath.Dir(c.certFile) instead of c.certFile as in Windows, so we can simplify this code.

Reply:
I want to do this specifically on windows alone, since any file directory changes are not needed on other platforms.